### DISCUSSION TITLE
Allow changing data types on calculated columns without first setting "Data Type Inferred" to false

### DISCUSSION THREAD

**Discussion by Stewwe1** [TabularEditor/TabularEditor3#1075] [Ideas]
As the title describes, I cannot change the data format in TE3 (it is grayed out), but in VS this is possible without any problems.

Very much looking forward to the solution :-)

Greetings stewwe

![TE3_Greyedout](https://user-images.githubusercontent.com/49725984/141083108-afcfbb1a-97e7-4b36-ab96-b6adec5f0517.PNG)
![VS_Possible](https://user-images.githubusercontent.com/49725984/141083111-241ab17c-058e-4f1c-a8c7-9d7f7a1ebff2.PNG)

**Comment by otykier** [comment]
Did you want to change the data type or the format string? These are two different properties.

**Comment by Stewwe1** [comment]
Hi Daniel!

I want to change the Data Type from date to decimal number

**Comment by otykier** [comment]
I see - I was just confused by your first screenshot where the "Date Format" property is highligted.

To change the data type of a calculated column you need to first set the "Data Type Inferred" property to false (I guess VS does this automatically, when you change the data type of a calculated column).

![image](https://user-images.githubusercontent.com/8976200/141098700-dc6c134a-d717-4c3e-b140-c665d85a3f8c.png)

**Comment by Stewwe1** [comment]
Ok, that works...but you have to know that ;-) Maybe it would be easier, if TE3 does this also automatically... just a suggestion.

Thank you for your help Daniel!
